### DISCUSSION TITLE
Few annoyances when launching terminal

### DISCUSSION THREAD

**Discussion by bogdanv-ntnu** [webmin/authentic-theme#1667] [General]
Anyone knows how to fix the white text background when text is pasted in the terminal window? (besides typing some stuff after)
It must be an override from chrome's user agent style since the theme doesn't set those values.
Also starting terminal doesn't activate the cursor in it.
I didn't notice these things in the past, must be recent problems.

**Comment by iliaross** [comment]
> Also starting terminal doesn't activate the cursor in it.

This seems like our bug!

> Anyone knows how to fix the white text background when text is pasted in the terminal window? (besides typing some stuff after)
> It must be an override from chrome's user agent style since the theme doesn't set those values.

Not sure what you mean? Can you show a screenshot?

**Comment by bogdanv-ntnu** [comment]
![image](https://github.com/webmin/authentic-theme/assets/73938209/9bff6c2d-3a7f-49f0-b848-0c613b563834)
and this is what i paste
![image](https://github.com/webmin/authentic-theme/assets/73938209/74438c74-799d-4d00-a3be-39e8c8f216a5)
and if i press enter with the text still highlighted the text will permanently remain highlighted. 
If i select the text again with the cursor then a translucent background highlighter will render the text visible.
![image](https://github.com/webmin/authentic-theme/assets/73938209/ab6ef64e-d1af-48d7-9287-c1dcbac2bbb0)

**Reply by iliaross** [reply]
Alright. The focus issue has been fixed in the latest Authentic theme commit. You can use theme configuration page to install latest update.

> and if i press enter with the text still highlighted the text will permanently remain highlighted.
> If i select the text again with the cursor then a translucent background highlighter will render the text visible.

I'm afraid there is not much we can do about it. It seems as upstream issue (Xterm.js rendering add-on). You can try disabling it by running in browser's console and reloading browser's page:

```
config_portable_module_xterm_render_addon = false;
theme.config.save();
```

**Comment by iliaross** [comment]
We may need to make rendering add-on optional from module configuration page.
